### PR TITLE
OSDOCS-6970-RN Azure NAT gateway

### PR DESCRIPTION
**Pull request by bscott-rh**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-6970

Link to docs preview:
https://63182--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-nat-gateway-azure

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by bscott-rh** [comment]
@jinyunma Hi Jinyun, PTAL at this release note for Azure NAT gateway support. Thank you

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -57,6 +57,12 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-14-installation-and-update"]
 === Installation and update
 
+[id="ocp-4-14-nat-gateway-azure"]
+==== Installing a cluster on Microsoft Azure using a NAT gateway (Technology Preview)
+
+In {product-title} {product-version}, you can install a cluster that uses a NAT gateway for outbound networking. This is available as a Technology Preview (TP). For more information, see xref:../installing/installing_azure/installation-config-parameters-azure.adoc#installation-configuration-parameters-additional-azure_installing-azure-customizations[Additional Azure configuration parameters].
```
Comment (kcarmichael08) [inline release_notes/ocp-4-14-release-notes.adoc:63]: Did you want the xref to go directly to the "Additional Azure configuration parameters" section? If so I think the xref needs to be `../installing/installing_azure/installation-config-parameters-azure#installation-configuration-parameters-additional-azure_installation-config-parameters-azure`

**Review by kcarmichael08 — COMMENTED** [review]
One question; otherwise lgtm!

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (bscott-rh) [reply release_notes/ocp-4-14-release-notes.adoc]: You're quite right, thanks

**Pull request merged**